state/aws/dynamodb/dynamodb.go

```diff
@@ -100,6 +105,17 @@ func (d *StateStore) Get(req *state.GetRequest) (*state.GetResponse, error) {
 		return nil, err
 	}
 
+	var ttl int64
+	if result.Item[d.ttlAttributeName] != nil {
```

Comment:
Careful, `meta.TTLAttribute` should not be required. So this means the value will be an empty string here. You cannot index into your item with an empty string.

I recommend doing the following:

```suggestion
        if d.ttlAttributeName != "" {
	if val, ok := result.Item[d.ttlAttributeName]; ok {
	   if err = dynamodbattribute.Unmarshal(val, &ttl); err != nil {
```